### DISCUSSION TITLE
java 7/8 too old

### DISCUSSION THREAD

**Discussion by ruben-nld** [gephi/gephi#2411] [Q&A] [answered]
I'm an avid Gephi user, but the security officers & IT architects won't allow Gephi on the company infrastructure. Main reason for them is the outdated java version that Gephi uses. Are there any plans for future releases to be compatible with newer Java versions?

**Comment by eduramiba** [comment, marked as answer]
We are working on a 0.9.3 release soon, you can read more in our blog: https://gephi.wordpress.com/2021/12/06/gephi-code-sustainability-retreat-2021-debriefing/

**Reply by ruben-nld** [reply]
I know, and I was excited to learn about this upcoming release! Can you say something about which JRE version will be used in the 0.9.3 release?

**Reply by eduramiba** [reply]
JDK 11, as seen at https://github.com/gephi/gephi/blob/master/pom.xml#L170
But we should consider using JDK 17 before release, as it's the current Java LTS

**Reply by ruben-nld** [reply]
Perfect! Yes, JDK 17 would be even better!

**Reply by mbastian** [reply]
I detected some JDK 17 issues related to HighDPI and Java2D so I sticked to JDK 11 for now. Once the 0.9.3 is out we can look into this again and see whether those issues were solved in the meantime.

**Reply by gwselke** [reply]
Also eagerly waiting for this -- our systems people just de-installed JDK 8 from my computer due to security concerns. 
I understand from the announcement (quoted above) that a JRE will be bundled with gephi so that it will be independent of locally installed versions. Is that still the plan?

**Reply by ruben-nld** [reply]
yes, you could try the latest nightly build: https://github.com/gephi/gephi#latest-releases
JRE is already bundled with gephi!

**Reply by gwselke** [reply]
Downloaded the nightly build of v0.9.3 as suggested, works like a charm -- many, many thanks, Ruben, for pointing me to this!

**Reply by mbastian** [reply]
Yes @gwselke that's the plan. No longer needed to install Java on your own as it's bundled with Gephi.

Glad the 0.9.3 build works out. Note that plugins are not yet working for that build.
